xml/storage_lvm.xml

```diff
@@ -1044,15 +1032,25 @@ root's password:
    </step>
   </procedure>
  </sect1>
+ <sect1 xml:id="sec-lvm-onboot">
+  <title>Disabling LVM on boot</title>
+
+  <para>
+   If there is an error on the LVM storage, the scanning of LVM volumes may
+   prevent entering the emergency/rescue shell. Thus further problem diagnosis
+   is not possible. To disable this scanning in case of a LVM storage failure,
```

Comment:
```suggestion
   prevent entering the emergency/rescue shell. Thus, further problem diagnosis
   is not possible. To disable this scanning in case of an LVM storage failure,
```